src/index.html

```diff
@@ -9,6 +9,65 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="selectors">
+      <div class="logo">
+
+      </div>
+      <nav>
+
+        <ul class="navigationBar">
+
+          <li class="navigationBar__element is-active">
+            <a class="navigationBar--first is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="navigationBar__element">
+            <a class="navigationBar--second">
+              Samsung
+            </a>
+          </li>
+
+          <li class="navigationBar__element">
+            <a class="navigationBar--third">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="navigationBar__element">
+            <a class="navigationBar--fourth" data-qa="hover">
```

Comment:
you don't need first, second, etc
use common class, then you can use nth-child if you need

Reply:
I ask here https://mate-parttime.slack.com/archives/C041JA5EJKV/p1663883029792849